zbus/src/abstractions/process.rs

```diff
@@ -94,3 +96,21 @@ where
 {
     Command::new(program).args(args).output().await
 }
+
+impl From<&Unixexec<'_>> for Command {
```

Comment:
This isn't exactly a conversion. `Unixexec` should provide a method to get the corresponding `Command`.

Reply:
True, although it was [previously argued](https://github.com/dbus2/zbus/pull/976#discussion_r1766701996) that this conversion should not live in `address::transport::unixexec` either (which makes sense too):

> [...] Though ::address is really standalone, so `crate::process::Command` part should be moved elsewhere. in ::process perhaps?

Another option would be to just move the logic into the `connection::socket::command::connect` function (which is the only place where this logic is needed).

In any case, it's fine by me either way so please let me know which one would be preferred:
a) move it into a method on `Unixexec` (with the caveat that it introduces some coupling between the `address` and `process` modules)
b) move into the `connection::socket::command::connect` method

PS: If the `a` method is preferred, any suggestion for the method name? I'm still somewhat new to Rust and I haven't familiarized myself with the naming conventions yet.

Reply:
Thanks, fixed in latest fixup commit (will do a final rebase once all review feedback is addressed).

Reply:
> will do a final rebase once all review feedback is addressed

I think I may have misunderstood this. You mean feedback addressed in #989? If so, I don't think you need to wait for that. You can already make the only change we need here as part of this PR: arg0 to be treated differently. We now have #1040 awaiting on this and both PRs now have conflicts with main so I can't run the CI on it to test it.

It's pitty that this PR got so delayed. There was a good reason I originally suggested that @elmarco should base his PR on yours, since this one is much less intrusive and simpler but oh well. :)

Reply:
> the only change we need here as part of this PR: arg0 to be treated differently.

I've fixed this in the latest revision. The only point still up for discussion is the implementation of `Command::for_unixexec` which is still a bit ugly, but other than that all remaining feedback has been addressed.